website/templates/project/addon/view_file.mako

```diff
@@ -41,11 +69,30 @@
 
 <%def name="javascript_bottom()">
     ${parent.javascript_bottom()}
+    % for script in tree_js:
+        <script type="text/javascript" src="${script | webpack_asset}"></script>
+    % endfor
     % if rendered is None:
         <script type="text/javascript">
             window.contextVars = window.contextVars || {};
             window.contextVars.renderURL = '${render_url}';
         </script>
         <script src=${"/static/public/js/view-file-page.js" | webpack_asset}></script>
+    % else:
+        <script type="text/javascript">
+        window.contextVars = $.extend(true, {}, window.contextVars, {
+                file: {
+                    name: '${file_name | js_str}',
+                    path: '${path | js_str}',
+                    provider: 'dataverse'
+                },
+                node: {
+                    urls: {
+                        files: '${urls['files'] | js_str}'
+                    }
+                }
+        });
+        </script>
+        <script src=${"/static/public/js/view-file-tree-page.js" | webpack_asset}></script>
```

Comment:
Why is this in the `else` block? Shouldn't this script be loaded even if rendered is not `None`?